#
# Studio gets confused by -pedantic.  This won't go upstream as newer versions
# change the compiler flag test and this will no longer be needed .
#

--- stunnel-4.56/configure.ac.orig	Mon Mar  9 12:33:03 2015
+++ stunnel-4.56/configure.ac	Mon Mar  9 12:33:20 2015
@@ -88,12 +88,6 @@
     [AC_MSG_RESULT([yes])],
     [AC_MSG_RESULT([no]); CFLAGS="$valid_CFLAGS"])
 
-AC_MSG_CHECKING([whether $CC accepts -pedantic])
-valid_CFLAGS="$CFLAGS"; CFLAGS="$CFLAGS -pedantic"
-AC_LINK_IFELSE([int main() {return 0;}],
-    [AC_MSG_RESULT([yes])],
-    [AC_MSG_RESULT([no]); CFLAGS="$valid_CFLAGS"])
-
 AC_MSG_NOTICE([**************************************** libtool])
 LT_INIT([disable-static])
 AC_SUBST([LIBTOOL_DEPS])
